core/arch/arm/tee/arch_tee_fs.c

```diff
@@ -49,7 +49,7 @@ int tee_fs_send_cmd(struct tee_fs_rpc *bf_cmd, void *data, uint32_t len,
 	thread_rpc_alloc_payload(sizeof(struct tee_fs_rpc) + len,
 				 &phpayload, &cpayload);
 	if (!phpayload)
-		goto exit;
+		return -1;
```

Comment:
Should `thread_rpc_alloc_payload()` and friends return error instead of relying on phpayload? May be in another PR?